Jenkinsfile_CNP

```diff
@@ -94,7 +95,8 @@ env.CASE_DOCUMENT_AM_URL = "http://ccd-case-document-am-api-aat.service.core-com
 env.RD_LOCATION_REF_API_BASE_URL = "http://rd-location-ref-api-aat.service.core-compute-aat.internal"
 env.RD_PROFESSIONAL_API_BASE_URL = "http://rd-professional-api-aat.service.core-compute-aat.internal"
 env.BEFTA_RESPONSE_HEADER_CHECK_POLICY="JUST_WARN"
-env.ELASTIC_SEARCH_FTA_ENABLED = "false"
+// TODO: remove TEMPORARY OVERRIDE to test GlobalSearch, reinstate: env.ELASTIC_SEARCH_FTA_ENABLED = "false"
+env.ELASTIC_SEARCH_FTA_ENABLED = "true"
```

Comment:
This should be reveresed if final test-def version changes are merged to `master`.